### ISSUE TITLE
Coordinate with the server to integrate the new moderation experience

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-dart#885]
### What
The new moderation tool is about to be deployed. Will update this as a backend is finalized

```[tasklist]
### API Endpoints for the new moderation experience
- [ ] POST /flags: Create a new flag for a product or image.
- [ ] GET /flags: Get a list of all flags.
- [ ] GET /flags/{flag_id}: Get a specific flag by its ID.
- [ ] POST /tickets: Create a new ticket for a product or image issue.
- [ ] GET /tickets: Get a list of all tickets.
- [ ] GET /tickets/{ticket_id}: Get a specific ticket by its ID.
- [ ] GET /tickets/{ticket_id}/flags: Get all flags associated with a specific ticket.
- [ ] PUT /tickets/{ticket_id}/status: Update the status of a ticket (open or closed).
```

### Data Models

- Flag: Represents a flag raised by a user on a product or image.
- Ticket: Represents a ticket associated with a flag, containing details about the issue.
- Additional Notes:

- The client should support validation of user input and error handling.
- Authentication mechanisms should be implemented to restrict access to specific functionalities.

PoC @raphael0202 + Valentin 

https://github.com/openfoodfacts/nutripatrol
https://github.com/openfoodfacts/nutripatrol/blob/main/app/api.py

### Part of
- openfoodfacts/smooth-app#4795

**Comment by teolemon** [comment]
@monsieurtanuki the API doc has just been deployed by @raphael0202

**Comment by monsieurtanuki** [comment]
@teolemon I guess it would be easy for me to replicate for nutripatrol what was done for prices, on the off-dart level.
Then Valentin could work on the Smoothie side.
@g123k Would that be ok with you or would you like Valentin to work on both off-dart and Smoothie for nutripatrol?

**Comment by monsieurtanuki** [comment]
@g123k ping

**Comment by monsieurtanuki** [comment]
Oh, I hadn't understood that nutripatrol couldn't be used as an API for the moment (e.g. "Login with your OpenFoodFacts account" when I click on https://nutripatrol.openfoodfacts.org/flag/product?barcode=3017620422003&source=web&flavor=off and there's no matching API `flag` entry).

**Comment by monsieurtanuki** [comment]
@raphael0202 Fair enough. I guess I clicked on the first link provided, which opened a HTML page.

**Comment by Valimp** [comment]
Could you assign me this issue please ?

**Comment by monsieurtanuki** [comment]
@raphael0202 I couldn't see in the [API docs](https://nutripatrol.openfoodfacts.org/api/docs) how to connect.
Is there a bearer token mechanism here, as in Prices?
